src/portal/lib/src/tag/tag-detail.component.ts

```diff
@@ -128,25 +109,60 @@ export class TagDetailComponent implements OnInit {
       : "TAG.ANONYMITY";
   }
 
-  public get highCount(): number {
-    return this._highCount;
+  get criticalCount(): number {
+    if (this.vulnerabilitySummary && this.vulnerabilitySummary.summary
+        && this.vulnerabilitySummary.summary.summary) {
+      return this.vulnerabilitySummary.summary.summary[VULNERABILITY_SEVERITY.CRITICAL];
+    }
+    return 0;
   }
 
-  public get mediumCount(): number {
-    return this._mediumCount;
+  get highCount(): number {
+    if (this.vulnerabilitySummary && this.vulnerabilitySummary.summary
+        && this.vulnerabilitySummary.summary.summary) {
+      return this.vulnerabilitySummary.summary.summary[VULNERABILITY_SEVERITY.HIGH];
+    }
+    return 0;
   }
 
-  public get lowCount(): number {
-    return this._lowCount;
+  get mediumCount(): number {
```

Comment:
Lots of duplicated code in these mediumCount(), lowCount, could you please abstract a common method and pass parameters to get different vulnerability count?

Reply:
Not that good to pass parameter. Because this properties is needed  in template file